package.json

```diff
@@ -15,15 +15,20 @@
     "@babel/core": "^7.11.1",
     "@babel/preset-env": "^7.11.0",
     "@rollup/plugin-babel": "^5.2.0",
+    "@types/webextension-polyfill": "^0.9.1",
```

Comment:
Should we move this into prod-dependency? What is the common practice for this kind of peer-dependency type?

Or we should just document it in readme?

Reply:
Or add a `"peerDependencies": {"types/webextension-polyfill": ">=0.9"}`?

Reply:
hmm, it seems that the standard practice for TypeScript libraries is to move all imported `@types/**` dependencies into prod-dependency... however, I'm against bloating the dist for something that won't be used for all users, only those who are using TS.
What if we kept it as a devDependency and just documented it to the readme, like this?

Installing via npm:
`npm install webext-launch-web-auth-flow`

Installing via npm with TypeScript support:
`npm install webext-launch-web-auth-flow @types/webextension-polyfill`

?